Packs/illuminate/Integrations/Analyst1/Analyst1.py

```diff
@@ -107,36 +116,121 @@ def return_outputs(self):
         }
 
         demisto.results(entry)
+        return
 
     def add_analyst1_context(self, key: str, data: Any):
         self.analyst1_context_data[key] = data
+        return
 
     def add_reputation_context(self, key: str, context: dict):
         self.reputation_context[key] = context
+        return
 
     def has_context_data(self):
         return len(self.analyst1_context_data) > 0
 
 
 class Client(BaseClient):
     def __init__(self, server: str, username: str, password: str, insecure: bool, proxy: bool):
-        # NB: 404 is a valid response since that just means no entries, and we want the UI to respect that and show "No Entries"
         super().__init__(
             base_url=f'https://{server}/api/1_0/',
             verify=not insecure,
             proxy=proxy,
-            auth=(username, password),
-            ok_codes=(200, 404)
+            auth=(username, password)
         )
 
     def indicator_search(self, indicator_type: str, indicator: str) -> dict:
         params = {'type': indicator_type, 'value': indicator}
         return self._http_request(method='GET', url_suffix='indicator/match', params=params)
 
+    def post_evidence(self, fileName: str, fileContent: str, fileEntryId: str,
+                      evidenceFileClassification: str, tlp: str, sourceId: str) -> dict:
+        # warRoomFileId: str, may want to be added in as a future capability
+        # but access from those files were inconsistent, so current scope is only content or file entry
+        data_to_submit = {
+            'evidenceFileClassification': evidenceFileClassification,
+            'tlp': tlp,
+            'sourceId': sourceId
+        }
+
+        evidence_to_submit = None
+        if fileContent is not None and fileContent and str(fileContent):
+            evidence_to_submit = {'evidenceFile': (fileName, str(fileContent))}
+        elif fileEntryId is not None and fileEntryId:
+            try:
+                filePathToUploadToA1 = demisto.getFilePath(fileEntryId)
+                evidenceOpened = open(filePathToUploadToA1['path'], 'rb')
+                # rb for read binary is default
+                evidence_to_submit = {'evidenceFile': (fileName, evidenceOpened.read())}
```

Comment:
```suggestion
                evidence_to_submit = {'evidenceFile': (fileName, evidenceOpened.read())}
                evidenceOpened.close()
```
Or use `with open` (recommended)

Reply:
This suggestion was applied via a separate commit so that testing could occur first.